x-pack/plugins/security_solution/server/lib/detection_engine/schemas/rule_converters.test.ts

```diff
@@ -206,27 +206,28 @@ describe('rule_converters', () => {
   });
 
   describe('convertPatchAPIToInternalSchema', () => {
-    test('should not update version for immutable rules', () => {
-      const patchParams = {
+    test('should set version to one specified in next params', () => {
```

Comment:
Could you add this as a new test instead and keep the existing tests around the automatic `version` update for the mutable and immutable cases?

Reply:
Added a separate test 👍